src/components/Config/Config.tsx

```diff
@@ -120,68 +146,148 @@ function Config() {
     }
   }, [checkConnection, wallet]);
 
+  const ArbitrumNetworkConnect = useCallback(async () => {
+    const ArbitrumTestChainId = '0x66eeb'; // arbitrum test net
+    if (typeof window.ethereum !== 'undefined') {
+      const chainId = await window.ethereum.request({ method: `eth_chainId` });
+      if (chainId === ArbitrumTestChainId) {
+        notification.success({ message: `You're connected to Automate!` });
+      } else {
+        console.log('oulalal, switch to the correct network,chainid: ' + chainId);
+        try {
+          await window.ethereum.request({
+            method: 'wallet_switchEthereumChain',
+            params: [{ chainId: ArbitrumTestChainId }],
+          });
+          console.log('You have succefully switched to Arbitrum Test network');
+        } catch (chainerror) {
+          if (chainerror.code === 4902) {
```

Comment:
This error code, while it's probably not something you need to extract to `constants.ts` since it's only used once here, it's still a good practice to declare it somewhere up top the function or even one line above with a descriptive name e.g. `const NETWORK_NOT_AVAILABLE_ERROR_CODE` and then use that. This way you have self-documenting code